incident_test.go

```diff
@@ -670,7 +670,7 @@ func TestIncident_SnoozeIncident(t *testing.T) {
 	var duration uint = 3600
 	id := "1"
 
-	err := client.SnoozeIncident(id, duration)
```

Comment:
It would be nice if we actually verified the `From` header was being sent here in the test. You can see https://github.com/PagerDuty/go-pagerduty/blob/0ef8f340dd3cab97a37f8149760b64a45510c036/incident_test.go#L958-L1007 as an example.

Reply:
Fixed